data/pools/transactionPool.go

```diff
@@ -667,6 +668,13 @@ func (pool *TransactionPool) recomputeBlockEvaluator(committedTxIds map[transact
 	}
 	pool.pendingBlockEvaluator, err = pool.ledger.StartEvaluator(next.BlockHeader, hint)
 	if err != nil {
+		var nonSeqBlockEval ledgercore.ErrNonSequentialBlockEval
+		if errors.As(err, &nonSeqBlockEval) {
+			if nonSeqBlockEval.EvaluatorRound <= nonSeqBlockEval.LatestRound {
+				pool.log.Infof("TransactionPool.recomputeBlockEvaluator: skipped creating block evaluator for round %d since ledger already caught up with that round", nonSeqBlockEval.EvaluatorRound)
+				return
+			}
+		}
```

Comment:
this looks not related to the change, has it come from another branch?

Reply:
No. The startevaluator has another special fail case.
This change add a special handling for that case.
( it's not a real error, and shouldn't be treated as such.. )

Reply:
Yes, I see, a new error from `StartEvaluator`, makes sense to have a special handling, thanks for pointing.